tests/features/common/kie-kieserver-common.feature

```diff
@@ -536,6 +536,15 @@ Feature: Kie Server common features
     And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value nodes on XPath //*[local-name()='cache-container']/*[local-name()='replicated-cache'][@name='nodes']/@name
     And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value jobs on XPath //*[local-name()='cache-container']/*[local-name()='replicated-cache'][@name='jobs']/@name
 
+  Scenario: Check KIE_SERVER_JBPM_CLUSTER cache module flag enabled
+    When container is started with env
+      | variable                        | value                |
+      | JGROUPS_PING_PROTOCOL           | kubernetes.KUBE_PING |
+      | KIE_SERVER_JBPM_CLUSTER         | true                 |
+    Then container log should contain KIE Server's cluster for Jbpm failover is enabled.
```

Comment:
please, match all the added content

Reply:
line 545 check <module name="org.infinispan" services="export"/> 546 checks for <module name="org.jgroups"/> no other content is added